packages/flutter/lib/src/widgets/app.dart

```diff
@@ -171,6 +178,11 @@ class WidgetsApp extends StatefulWidget {
   /// Used by `showPerformanceOverlay` observatory extension.
   static bool showPerformanceOverlayOverride = false;
 
+  /// If true, forces the widget inspector to be visible.
+  ///
+  /// Used by `showWidgetInspector` debugging extension.
```

Comment:
done. fixed all the existing cases where "the" was omitted to bring in compliance with Hixie english. :)